controllers/administration.js

```diff
@@ -1165,13 +1165,19 @@ const getUsersWithoutConsent = async (req, roleName, classId) => {
 		users = (await api(req).get('/users', { qs, $limit: false })).data;
 	}
 
-	const consents = (await api(req).get('/consents', {
-		qs: {
-			userId: { $in: users.map(u => u._id) },
-			$populate: 'userId',
-			$limit: false,
-		}
-	})).data;
+	let consents = [];
+	const batchSize = 50;
```

Comment:
Das könnte ein API helper werden.

Reply:
auf jeden Fall. Die Frage ist jetzt nur, ob das jetzt noch sinnvoll ist, wenn mehr und mehr in den Server umgezogen wird. Ich kann es mir auf jeden Fall als Folgeticket vorstellen.